install.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+# curl https://github.com/srevinsaju/zap/raw/main/install.sh | sh
+
+
+# A Bash Script to auto fetch latest release from srevinsaju/zap releases
+
+echo
+echo "####################"
+echo
+echo  '~   ZAP Installer  ~'
+echo
+echo "####################"
+echo
+
+
+
+
+# Config
+REPO="srevinsaju/zap"
+
+
+# Clean
+if [ -f "./zap" ]; then
+	echo [~] Found old zap binary. Removing....
+	rm "./zap"
+fi
```

Comment:
Is it correct that we are using the current directory as a place to temporarily download `zap`?

Reply:
Yes. We temporarily download it to current directory and then move it depending on script user. If script is running as root we move to system bin folder.

Reply:
Should i directly download zap to bin folders?

Reply:
If its possible, I think that would be better, or alternatively, maybe download to `/tmp` or something like that I think, or even `~/.cache/zap/...`

Reply:
Whichever is easier ^

Reply:
Sure. The script was no root install in the first so i quickly hacked it to support root installations. Thats why it uses the current directory.

I'll make sure to download to tmp.